### PR TITLE
Ltd 2470 notes and timelines styling

### PR DESCRIPTION
**Pull request by kevincarrogan**

### CODE REVIEW

File: caseworker/activities/views.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        if not settings.FEATURE_FLAG_NOTES_TIMELINE_2_0:
+            raise Http404("Feature flag disabled")
+        return super().dispatch(*args, **kwargs)
+
+    @cached_property
+    def case_id(self):
+        return str(self.kwargs["pk"])
+
+    @cached_property
+    def case(self):
```
Comment (depsiatwal) [inline caseworker/activities/views.py:24]: don't we have mixin for some of this stuff 
probably not generic enough

File: caseworker/advice/templates/advice/view-advice.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
 {% load crispy_forms_tags %}
 
 {% block header_tabs %}
-	{% include "advice/case-tabs.html" with case=case queue=queue %}
+	{% include "includes/case-tabs.html" with case=case queue=queue selected_tab="advice" %}
```
Comment (depsiatwal) [inline caseworker/advice/templates/advice/view-advice.html:7]: is this existing template do we need to feature flag here ?

File: caseworker/advice/templates/advice/view_countersign.html [hunk 3]
```diff
@@ -4,7 +4,7 @@
 {% load crispy_forms_tags %}
 
 {% block header_tabs %}
-	{% include "advice/case-tabs.html" with case=case queue=queue %}
+	{% include "includes/case-tabs.html" with case=case queue=queue selected_tab="advice" %}
```
Comment (depsiatwal) [inline caseworker/advice/templates/advice/view_countersign.html:7]: same as above FF ?

**Review by depsiatwal — COMMENTED** [review]
LGTM some minor comments which may not be applicable

File: caseworker/advice/templates/advice/view-advice.html (hunk 2 above)
Comment (kevincarrogan) [reply caseworker/advice/templates/advice/view-advice.html:7]: This is me just re-using and moving an include that already exists in multiple places. It doesn't require any feature flags.

File: caseworker/activities/views.py (hunk 1 above)
Comment (kevincarrogan) [reply caseworker/activities/views.py:24]: I contemplated trying to extract this out as a mixin. There are others mixins that have this but they also include other functionality and I didn't want to have to rewrite a whole bunch of mixins and class hierarchies just for this.

**Review by depsiatwal — APPROVED** [review]
LGTM

**Pull request merged**